static/js/stream_pill.js

```diff
@@ -0,0 +1,51 @@
+function get_user_ids_from_sub(items) {
+    let user_ids = [];
+    const stream_ids = items.map((item) => item.stream_id);
+    for (const stream_id of stream_ids) {
+        const sub = stream_data.get_sub_by_id(stream_id);
+        if (!sub) {
+            continue;
+        }
+        const subscribers = stream_edit.get_users_from_subscribers(sub.subscribers);
+        user_ids = user_ids.concat(subscribers.map((user) => user.user_id));
+    }
+    return user_ids;
+}
+
+exports.get_user_ids = function (pill_widget) {
+    const items = pill_widget.items();
+    let user_ids = get_user_ids_from_sub(items);
+    user_ids = Array.from(new Set(user_ids));
+
+    user_ids = user_ids.filter(Boolean);
+    return user_ids;
+};
+
+exports.append_stream = function (stream, pill_widget) {
+    pill_widget.appendValidatedData({
+        display_value: "#" + stream.name + ": " + stream.subscriber_count + " users",
+        stream_id: stream.stream_id,
+    });
```

Comment:
When displaying these pills, do we want to show the number of users being added?  I think that'd be a very helpful visual hint.

Reply:
Oh, I see, this is in a later commit.  OK.